frontend/src/app/modules/auth/sign-up/sign-up.component.ts

```diff
@@ -89,7 +91,9 @@ export class SignUpComponent extends BaseComponent implements OnInit {
 
     private setUpData() {
         this.countries = this.countriesTzLangProvider.getCountriesList().map((x) => x.name);
-        this.languages = this.countriesTzLangProvider.getLanguagesList();
+        this.constantsService.getAllLanguages().subscribe((languages) => {
```

Comment:
Can we save it to localStorage and then check if there are languages there, use them, if not - load

Reply:
this logic is implemented here
frontend/src/app/core/services/constants.service.ts